lib/ansible/plugins/doc_fragments/connection_persistent.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+class ModuleDocFragment(object):
+
+    # Standard files documentation fragment
+    DOCUMENTATION = r'''
+options:
```

Comment:
```suggestion
options:
  persistent_connect_timeout:
    type: int
    description:
    - Configures, in seconds, the amount of time to wait when trying to initially
      establish a persistent connection.  If this value expires before the connection
      to the remote device is completed, the connection will fail.
    default: 30
    ini:
    - section: persistent_connection
      key: connect_timeout
    env:
    - name: ANSIBLE_PERSISTENT_CONNECT_TIMEOUT
    vars:
    - name: ansible_connect_timeout
```

Reply:
Actually, on the whole, this belongs in netcommon, so now it will be: https://github.com/ansible-collections/ansible.netcommon/pull/308/files#diff-e84f32f4c9057b4900c0593346d486cf66db31cb92dcf9016d75d0bba03ce88d

Reply:
ok, so no need to add this here now? have the rest of the PRs been merged and this will pass your tests now?

Reply:
Having had some time to reflect on it, this should probably stay here and get updated while `module_utils/connection.py` and friends are in core.

We want to maintain the ability to make persistent connections that don't depend on `ansible.netcommon`. `httpapi` for example could theoretically live somewhere else and not have much issue. A complete version of this doc_fragment has been merged to netcommon, but that really is only because this is pretty useful but we would not be able to use it for a long time if it were just in core.

Basically, my opinion is that while it remains possible to make a standalone persistent connection without any collections, (which is pretty much just needs `ansible-connection` & `connection.py`), then this doc_fragment should probably live here, too.

Reply:
added